lib/galaxy/config/sample/file_sources_conf.yml.sample

```diff
@@ -2,7 +2,7 @@
   id: dropbox1
   label: Dropbox files (configure access in user preferences)
   doc: Your Dropbox files - configure an access token via the user preferences
-  accessToken: ${user.preferences['dropbox|access_token']}
+  access_token: ${user.preferences['dropbox|access_token']}
```

Comment:
Can you please handle `accessToken` as a fallback on line 26 of lib/galaxy/files/sources/dropbox.py